### DISCUSSION TITLE
EF Core support for Maybe<> type

### DISCUSSION THREAD

**Discussion by alexmurari** [vkhorikov/CSharpFunctionalExtensions#416] [General]
Just for the knowledge of the community, there's an open issue in the EF Core repo requesting support for the `Maybe<>`  type for navigation properties.

https://github.com/dotnet/efcore/issues/22504

Since it's a thumbs up system, don't forget to go there and leave a 👍🏻

**Comment by vkhorikov** [comment]
Sounds good! 👍🏻
